docs/src/main/sphinx/connector/postgresql.rst

```diff
@@ -93,6 +93,34 @@ catalog named ``sales`` using the configured connector.
 Type mapping
 ------------
 
+The data type mappings are as follows:
+
+=================== ================================ =======================================================================
+PostgreSQL          Trino                            Notes
+=================== ================================ =======================================================================
+``BIT``             ``BOOLEAN``
```

Comment:
PostgreSQL has a real boolean type too.

Also do we properly handle `bit(n)` in the connector?

Reply:
> Also do we properly handle bit(n) in the connector?

I don't think so.

Reply:
https://github.com/trinodb/trino/issues/11384